README.md

```diff
@@ -47,7 +47,8 @@ There are currently eight samples that are available in each language supported:
 * __insert_data_using_expressions__
   * This example shows how you can use SQL expressions in Hyper API Inserter to transform or compute data on the fly during data insertion
 
-
+* __defragment_data_of_existing_hyper_file__
```

Comment:
Can we please add the "Community supported" tag?

Reply:
Just this sample? Or all samples (at the repo level)?

Reply:
I added it to just this sample. @tsjoblad  @jonas-eckhardt  I'm not sure how you were planning to split the community supported samples from the Tableau supported ones. I just threw the label in here for this one.

Reply:
Please add it at the sample level. We will put each of them in individual folders, each with its own readme + support level.

@tsjoblad, right?